client/homescreen/layout.js

```diff
@@ -148,9 +168,21 @@ export default compose(
 		const { isUndoRequesting } = getUndoDismissRequesting();
 		const { getOption, isResolving } = select( OPTIONS_STORE_NAME );
 
+		const welcomeModalDismissed =
+			getOption( 'woocommerce_task_list_welcome_modal_dismissed' ) ===
+			'1';
+
+		const welcomeModalDismissedIsResolving = isResolving( 'getOption', [
```

Comment:
@joshuatf thats kind of interesting because at least in local dev testing I saw this option returning `undefined` 🤔 I might need to revisit this.

I agree we need to come up with some patterns for handling options and after your feedback on that post, I'm thinking about what we can do here.

Reply:
Since this is safer for now as you say, I'm gonna leave it as is, since it doesn't cost much verbosity to be safer.